screencapturepermissions.m

```diff
@@ -0,0 +1,36 @@
+#import <Foundation/Foundation.h>
```

Comment:
The filename should be `screen-capture-permissions.m`.

Reply:
Done, thanks for the suggestion